src/_fixtures.ts

```
@@ -0,0 +1,245 @@
+			}),
+		);
+	}
+
+	return createFixture({ projects });
+}
+
+/**
+ * Creates a fixture with raw JSONL content (including invalid lines)
+ */
+export async function createRawJSONLFixture(project: string, sessionFile: string, content: string): Promise<FsFixture> {
+	return createFixture({
```

ESLint

[error] Unsafe call of a(n) `error` type typed value.

(ts/no-unsafe-call)